src/stories/alert.stories.js

```diff
@@ -48,6 +52,13 @@ const Template = (args) => {
   alert.setAttribute('data-icon-size', args.iconSize);
   alert.setAttribute('data-extra-classes', args.extraClasses);
   alert.setAttribute('data-background-color', args.backgroundColor);
+
+    if (args.isOpen) {
+      alert.setAttribute('is-open', true);
+    } else {
+      alert.removeAttribute('is-open');
+    }
```

Comment:
In terms of code structure, this is 100%. We create a storybook arg that then controls the presence of an attribute on the custom element. 

My only nit is I think we should name the arg and attribute `closeable` instead of `is-open`. The attribute should control when there is a close button on the alert, not whether the alert is open or not.

Reply:
That's a perfectly fine nit. I was honestly not sure how to name I just didn't want to bug you lol